metadata-service/services/src/main/java/com/linkedin/metadata/recommendation/candidatesource/TopPlatformsSource.java

```diff
@@ -18,18 +27,48 @@
 public class TopPlatformsSource extends EntitySearchAggregationSource {
 
   /**
-   * TODO: Remove this once we permit specifying set of entities in aggregation API (filter out assertions)
+   * Set of entities that we want to consider for defining the top platform sources.
+   * This must match SearchUtils.SEARCHABLE_ENTITY_TYPES
    */
-  private static final Set<String> FILTERED_DATA_PLATFORM_URNS = ImmutableSet.of(
-      "urn:li:dataPlatform:great-expectations"
+  private static final Set<String> SEARCHABLE_ENTITY_TYPES = ImmutableSet.of(
+      Constants.DATASET_ENTITY_NAME,
+      Constants.DASHBOARD_ENTITY_NAME,
+      Constants.CHART_ENTITY_NAME,
+      Constants.ML_MODEL_ENTITY_NAME,
+      Constants.ML_MODEL_GROUP_ENTITY_NAME,
+      Constants.ML_FEATURE_TABLE_ENTITY_NAME,
+      Constants.ML_FEATURE_ENTITY_NAME,
+      Constants.ML_PRIMARY_KEY_ENTITY_NAME,
+      Constants.DATA_FLOW_ENTITY_NAME,
+      Constants.DATA_JOB_ENTITY_NAME,
+      Constants.GLOSSARY_TERM_ENTITY_NAME,
+      Constants.GLOSSARY_NODE_ENTITY_NAME,
+      Constants.TAG_ENTITY_NAME,
+      Constants.ROLE_ENTITY_NAME,
+      Constants.CORP_USER_ENTITY_NAME,
```

Comment:
let's actually remove user, group, role tag, glossary term, glossary node, domain, data product

since all of these won't be source of external domains anyways